alignak_backend/models/host.py

```diff
@@ -1079,18 +1072,25 @@ def get_schema():
         },
         'schema_deleted': {
             'ls_impact': {
-                'schema_version': 1,
+                'schema_version': 2,
```

Comment:
Why update this number?

Reply:
Because this firled was removed with the schema_version = 2